conf/application.conf

```diff
@@ -34,7 +34,7 @@ play.http.errorHandler = "uk.gov.hmrc.play.bootstrap.backend.http.JsonErrorHandl
 # Play Modules
 # ~~~~
 # Additional play modules can be added here
-
+play.modules.enabled += "uk.gov.hmrc.play.bootstrap.AuthModule"
 
 # The application languages
```

Comment:
The references to the hello world controller can be removed (line 61)

Reply:
removed references to the controller